lib/maverick/exception.ex

```diff
@@ -17,7 +17,7 @@ defmodule Maverick.Exception.Default do
           |> Jason.encode!()
 
         conn
-        |> Plug.Conn.put_resp_content_type("application/json", nil)
+        |> Plug.Conn.put_resp_content_type("application/json")
```

Comment:
Not sure, I remember adding the `nil` argument to appease the way we were testing the `content-type` header.  After updating the testing code, I assumed I should remove the argument.  I can add it back if you think it will cause a problem.